src/components/SecondaryNav/SecondaryNavHeader/SecondaryNavHeader.scss

```diff
@@ -0,0 +1,10 @@
+.secondaryNavHeader {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    min-height: 3em;
```

Comment:
I think we should add a min-height on `secondaryNav_header`

Reply:
I think that in that case what I put in the header will not be centered and aligned no?

Reply:
unless I do height: 100%, I don't know which is better